src/main/java/io/skai/okta/internshipstreamingserviceparserxml/services/converter/EpisodeAdditionalInfoConverter.java

```diff
@@ -0,0 +1,19 @@
+package io.skai.okta.internshipstreamingserviceparserxml.services.converter;
+
+import io.skai.okta.internshipstreamingserviceparserxml.dto.EpisodeAdditionalInfo;
+import io.skai.okta.internshipstreamingserviceparserxml.dto.ImdbJsonItem;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EpisodeAdditionalInfoConverter {
+    public EpisodeAdditionalInfo convert(long episodeId, ImdbJsonItem imdbJsonItem) {
+        return EpisodeAdditionalInfo.builder()
+                                    .id(episodeId)
```

Comment:
episode id should be separate column, you can name it `parent_id` for example, but id itself should be autoincrement of the table